.pre-commit-config.yaml0

```diff
@@ -0,0 +1,163 @@
+exclude: |
+  (?x)
+  # NOT INSTALLABLE ADDONS
+  # END NOT INSTALLABLE ADDONS
+  ^l10n_br_account/|
```

Comment:
Hello @rpsjr , your module looks great however I think you are bringing some meta-files from the Brazilian localization that shouldn't be included in this PR. This is the case with this .pre-commit-config.yaml0, .isort.cfg, .coveragerc, .editorconfig0 files at least. Could you amend this PR (or better with rebase -i) to remove these files please?